mpxe/sims/mci.py

```diff
@@ -0,0 +1,47 @@
+from mpxe.protogen import stores_pb2
+from mpxe.protogen import mcp2515_pb2
+from mpxe.protogen import gpio_pb2
+
+import time
+
+from mpxe.sims import sim
+
+GPIO_KEY = (stores_pb2.MxStoreType.GPIO, 0)
```

Comment:
In the absence of the utility modules/classes I described a couple comments ago, I think Sim should at least provide a `get_gpio_state` function to avoid having to mess with the store to get gpio states.